.gitlab-ci.yml

```diff
@@ -43,10 +43,6 @@ Build core-image-minimal:
   variables:
     TEST_BUILD_DIR: 'build-core-image-minimal'
     BITBAKE_TARGETS: 'core-image-minimal'
-  artifacts:
```

Comment:
This could have been the other way around: adding these artifacts to other branches. But I don't think anybody is really using that right now (and it's a little bit tricky as the image needs to be provisioned externally), so it should be fine to remove it for the sake of uniformity.

Reply:
Yeah, I don't even know how to get/use those myself. Could be interesting, though, but if it becomes necessary later we can always re-add it later.